app/workers/users/bust_cache_for_space_change_worker.rb

```diff
@@ -0,0 +1,14 @@
+module Users
+  # Conditionally delete the async cache when a Space changes
+  #
+  # @note Good news for those of you using Redis as your cache store, you have access to `Rails.cache.delete_matched`
+  #
+  # @see https://api.rubyonrails.org/classes/ActiveSupport/Cache/RedisCacheStore.html#method-i-delete_matched
+  class BustCacheForSpaceChangeWorker < BustCacheBaseWorker
+    def perform
+      return unless Rails.cache.respond_to?(:delete_matched)
+
+      Rails.cache.delete_matched("^#{AsyncInfoController::ASYNC_INFO_CACHE_KEY_PREFIX}-")
```

Comment:
@maestromac this is exactly the context I needed, and I think for now we will not need this (because we aren't going to enable the feature on DEV).  However, we now have this in our toolbox.